setup.py

```diff
@@ -11,7 +11,9 @@
     author='Vincent Driessen',
     author_email='me@nvie.com',
     description=__doc__,
-    packages=find_packages(exclude='tests'),
```

Comment:
I just saw you updated the version in pypi. I'll test again and inform you in a few minutes